lib/httpInterface.js

```diff
@@ -342,6 +360,24 @@ function handleGetList(request, reply) {
       reply.getHttpError(INTERNAL_SERVER_ERROR_STATUS_CODE, error.message || 'something went wrong')
     })
     .pipe(this.castResultsAsStream())
+    .pipe(through2.obj((chunk, _, callback) => {
+      const lookupPaths = JSONPath({ json: chunk, resultType: 'pointer', path: '$..[?(@ !== null && @ !== undefined && @.label && @.value)]' })
+        .map(path => path
+          .split('/')
+          .filter(str => str.length > 0)
+          .join('.')
+        )
```

Comment:
Looks like you want to remove consecutive backslashes and replace them with a single .

If so, I think that you can achieve with the following:
```suggestion
      const lookupPaths = JSONPath({ json: chunk, resultType: 'pointer', path: '$..[?(@ !== null && @ !== undefined && @.label && @.value)]' })
        .map(path => path.replace(/\\+/g, '.')
```

`replace` with a regex is more readable and quicker.